docs/en/dgs/dataloader.md

```diff
@@ -1,3 +1,158 @@
-# Load data from source
+# Loading From Your Data Source
 
-(comming soon...)
+By using kafka queues as the entry for streaming graph updates, the Dynamic-Graph-Service can decouple the different
+data sources and sampling workers.
+you should develop your own dataloader to read you data source and produce the graph updates into output kafka queues
+for further consuming.
+
+## Rules to Follow
+
+When you process you source data and produce them in your dataloader, you should follow these rules:
+- The graph schema from your source data must be consistent with dgs.
+- A produced kafka message must be a batch of graph updates, defined as a flatbuffers table [RecordBatchRep](https://github.com/alibaba/graph-learn/blob/master/dynamic_graph_service/fbs/record.fbs).
+- Graph updates and partitioned for sampling workers, all records in one batch must have the same data partition id.
+
+## Dataloader SDK
+
+In practice, it is difficult for users to batch records with dgs graph schema, manage partitioning logic and
```

Comment:
some time "dgs", sometimes "Dynamic-Graph-Service".

Reply:
Explain that this is an abbreviation at the beginning, and use dgs uniformly later